src/main/java/org/folio/dew/repository/FTPObjectStorageRepository.java

```diff
@@ -122,11 +130,8 @@ private void disconnect(FTPClient ftpClient) {
     }
   }
 
-  private static String getReplyMessage(Integer replyCode, String replyMessage) {
-    if (replyCode == 550) {
-      return FILE_UPLOAD_FAILED + "Please check if user has write permissions";
-    }
-    return FILE_UPLOAD_FAILED + replyMessage;
+  private static FtpException getFtpException(FTPClient ftpClient, String errorPrefix) {
```

Comment:
I dont know why it was static previosly, lets make it as non static, because I don't see reasons to remain it as static